package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maskbook",
-  "version": "1.21.3",
+  "version": "1.21.4",
```

Comment:
Did you change it manually?

Reply:
There is no need to change this field from now on. Please remove this version field. We'll use `pacakges/maskbook/src/manifest.json` as the source of truth